### PR TITLE
GML-1613 support continuous conversation P1

### PR DESCRIPTION
**Pull request by luzhoutg**
1. add new endpoint for continuous conversation support, can be merged with /{graphname}/query in the future when it's stable
2. modify the prompts to use conversation history for coreference resolution if needed
3. add unit test on Demo_Graph1 
4. will add metadata support (TS, feedbacks) in another PR

### CODE REVIEW

File: copilot/app/routers/inquiryai.py [hunk 1]
```diff
@@ -210,6 +210,195 @@ def retrieve_answer(
 
     return resp
 
+conversation_history = []
```
Comment (parkererickson-tg) [inline copilot/app/routers/inquiryai.py:213]: Does this global variable work for multiple users at the same time?

File: copilot/app/routers/inquiryai.py [hunk 2]
```diff
@@ -210,6 +210,195 @@ def retrieve_answer(
...
+
+# TODO: This could be merged with /{graphname}/query endpoints, all agents can be refactored in seperated function or file
```
Comment (parkererickson-tg) [inline copilot/app/routers/inquiryai.py:215]: Why couldn't this be the same endpoint?

**Comment by RobRossmiller-TG** [comment]
Thoughts on merging this into the ui_integration branch? I'll need this code anyway to connect continuous conversations into the UI

**Comment by parkererickson-tg** [comment]
> Thoughts on merging this into the ui_integration branch instead of dev? I'll need this code anyway to connect continuous conversations into the UI

This seems to me like a good idea.

File: copilot/app/routers/inquiryai.py (hunk 1 above)
Comment (luzhoutg) [reply copilot/app/routers/inquiryai.py:213]: Ideally, we need to store the history for different users separately. 
user_conversation_history: Dict[str, List[Dict[str, str]]] = {}

File: copilot/app/routers/inquiryai.py (hunk 2 above)
Comment (luzhoutg) [reply copilot/app/routers/inquiryai.py:215]: We can use the same endpoint. Was just considering to merge it after getting more stable results. I will merge it to use the same endpoint now.

**Pull request merged**